app/views/vehicles/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1>Vehicles</h1>
-
+<p><a href="/vehicles/new" id="creation link">Create a new vehicle!<a><p>
```

Comment:
There is a link_to helper method that you should be aware of. Check out this [article](https://www.rubyguides.com/2019/05/rails-link_to-method/). I think this plain html hyperlink works fine but just fyi

Reply:
I didn't know about that.  I'll be sure to implement it!